packages/client/modules/email/components/SummaryEmail/ExportToCSV.tsx

```diff
@@ -1,5 +1,6 @@
 import graphql from 'babel-plugin-relay/macro'
-import Parser from 'json2csv/lib/JSON2CSVParser' // only grab the sync parser
+import type {Parser as JSON2CSVParser} from 'json2csv'
```

Comment:
i don't usually import type, but i wanted to be sure we aren't imported the whole package because it's a big one